presto-product-tests/conf/docker/files/run-tempto.sh

```diff
@@ -2,6 +2,25 @@
 
 set -euo pipefail
 
+# If we have an overriden JDK volume mount, use it
+# This is set to /dev/null ignore
+if [ -d /docker/volumes/overridejdk ]; then
+  CONTAINER_JAVA_HOME=$JAVA_HOME
+  export JAVA_HOME=/docker/volumes/overridejdk
+  export PATH=$JAVA_HOME/bin:$PATH
+fi
+
+echo "Starting TemptoProductTestRunner with java set to :"
+java -version
+
+# Check if Java version is 17
+# This relies on the version string adhering to the format "17.x.xx"
+#JAVA_VERSION=$(java -version 2>&1 | awk -F '"' '/version/ {print $2}' | cut -d'.' -f1)
+#if [ "$JAVA_VERSION" == "17" ]; then
+#    echo "Java version is 17, setting custom JVM config"
+#    JVM_CONFIG="${PRESTO_CONFIG_DIRECTORY}/jvm17.config"
```

Comment:
The separation of JVM configs we added in #a3843f67 for the presto-launcher-wrapper can be undone now. I'm okay to punt this to a later PR